Hackney.Shared.CautionaryAlerts/Factories/EntityFactory.cs

```diff
@@ -133,7 +133,9 @@ public static PropertyAlertNew ToDatabase(this CreateCautionaryAlert entity)
                 Code = entity.Alert.Code,
                 CautionOnSystem = entity.Alert.Description,
                 DateOfIncident = entity.IncidentDate.ToString("d", CultureInfo.InvariantCulture),
-                Reason = entity.IncidentDescription
+                Reason = entity.IncidentDescription,
+                IsActive = IsActive,
+                AlertId = AlertId,
```

Comment:
This `ToDatabase` method appears to be **untested**.

It would be good to add a few tests.